installers/Windows/README.md

```diff
@@ -41,4 +41,56 @@ https://github.com/mu-editor/mu/blob/master/win_installer.py
 
 Files covered:
 
-installer.py
\ No newline at end of file
+installer.py
+
+## Step by Step: Creating a Spyder Windows Installer
+To create a Spyder Windows Installer you must follow the following steps.
+
+### Creating a Conda Environment 
+- Download and install Anaconda. You can find the latest version [here](https://www.anaconda.com/products/distribution). 
+- Open the Anaconda prompt and make sure that you have admin permissions.
+- Create the new enviroment with the following commands:
+bash
+$ conda create -n installer python=3.8.10
+$ conda activate installer
+
+The first command creates an enviroment called "installer" with Python version 3.8.10 and the second one, changes the current enviroment to the new one, "installer".'
+
+After changing to the new enviroment, install yarg with pip. 
+bash
+pip install yarg
+
+### Creating Enviroment Variables
+Before running any script, you must set some enviroment variables first. 
+The list below shows all the variables that must be created.
+-   ASSETS_URL=https://github.com/spyder-ide/windows-installer-assets/releases/download/0.0.7/assets.zip
+-   UNWANTED_PACKAGES=pip spyder-kernels python-slugify Rtree QDarkStyle PyNaCl
+-   SKIP_PACKAGES=bcrypt slugify
+-   ADD_PACKAGES=spyder_kernels blib2to3 _black_version blackd rtree qdarkstyle nacl
+-   PYNSIST_REQ=pynsist==2.7
+-   EXE_NAME=Spyder_64bit_full.exe
+
+For each variable on the list, type
+`set <variable=value>` in the anaconda prompt. 
+### Running the Installer Script
+Type the following line into the anaconda command prompt:
```

Comment:
Probably we should add here that the current working directory is the repository root directory for the command bellow works